src/filesys.cpp

```diff
@@ -305,7 +305,7 @@ bool RecursiveDelete(const std::string &path)
 		char argv_data[3][10000];
 		strcpy(argv_data[0], "/bin/rm");
 		strcpy(argv_data[1], "-rf");
-		strncpy(argv_data[2], path.c_str(), 10000);
+		memcpy(argv_data[2], path.c_str(), 10000);
```

Comment:
yeah, no
you are copying 10k bytes from a buffer that's probably smaller

if you want to avoid the warning this could use `snprintf`